x-pack/plugins/reporting/server/export_types/printable_pdf_v2/index.ts

```diff
@@ -9,7 +9,6 @@ import {
   LICENSE_TYPE_ENTERPRISE,
   LICENSE_TYPE_GOLD,
   LICENSE_TYPE_PLATINUM,
-  LICENSE_TYPE_STANDARD,
```

Comment:
Same as above: this needs to remain for Cloud (unless something has changed)

Reply:
I don't think anything has changed! I intended to add a comment on one of these lines asking whether these can be removed, I assumed they were `basic`. I'll add an inline comment stating that they are used for Cloud, thanks!